README.md

```diff
@@ -56,6 +56,20 @@ All html patterns in your pattern library will be located in `./patterns`. They
 
 Any changes to files in the `./patterns` directory will automatically be imported into your local Pattern Lab instance as long as you have run the `gulp serve` command and your server is running.
 
+## Can I include one pattern in another?
+
+as a twig include, yes.
+
+ * [how to twig includes](http://twig.sensiolabs.org/doc/tags/include.html)
+ * [example pattern with include](https://github.com/pattern-library/pattern-library/blob/master/patterns/molecules/media/figure-image/figure-image.twig)
+
+## Can I modify an existing pattern?
```

Comment:
@rebmullin can you change this to something like

"Can I modify a pattern imported from an NPM-based pattern library?"

Or something that would be more clear to show you mean patterns inside `node_modules`?
(that is what you mean, right?)